spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  describe 'date formatting constants' do
+    context 'FULL_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+
+    context 'MONTH_YEAR_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+  end
+
+  context ':format_full_date' do
+    it 'formats a date' do
```

Comment:
I don't share your worry about `.strftime` suddenly changing from under us. I do agree with you on what we are actually testing though. This reminds me of tests that check a certain function have been called, in a slightly different format and seems tightly coupled to its implementation.
I would expect to see a test along the format of.
- Create a Date object  with a specific day, month, year
- check its format matches what we expect for that date